ingestion/functions/parsing/japan/japan.py

```diff
@@ -43,23 +41,44 @@ def convert_location(raw_entry):
         if term and term != "Unspecified"]
     return {"query":  ", ".join(query_terms)}
 
-
 def detect_notes(raw_notes):
     if "notes" in raw_notes:
         return raw_notes["notes"]
     else:
         return None
 
-
-def convert_date(raw_date: str):
+def convert_date(raw_date: Dict):
     """
     Convert raw date field into a value interpretable by the dataserver.
-
+​
     The date filtering API expects mm/dd/YYYYZ format.
     """
-    date = datetime.strptime(raw_date, "%Y-%m-%d")
+    date = datetime.strptime(raw_date["dateAnnounced"], "%Y-%m-%d")
```

Comment:
If you wanted, you could have the function accept entry["dateAnnounced"] as a string, rather than passing in the whole entry. That way, you could reuse this function to process death_date rather than having to repeat the logic. It looks like it works fine as is, though.